Tasks/ANTPreview/task.json

```diff
@@ -78,52 +78,47 @@
             "helpMarkDown": "Test results files path.  Wildcards can be used.  For example, `**/TEST-*.xml` for all xml files whose name starts with TEST-."
         },
         {
-            "name": "publishCodeCoverageResults",
-            "type": "boolean",
-            "label": "Publish to VSO/TFS",
-            "required": true,
-            "defaultValue": "false",
-            "groupName": "codeCoverage",
-            "helpMarkDown": "Select this option to publish Code Coverage results produced by the Maven build to VSO/TFS.Code Coverage Summary and code Coverage Files would be published to TFS Server"
-        },
-        {
             "name": "codeCoverageTool",
             "type": "pickList",
-            "label": "CodeCoverage Tool",
+            "label": "Code Coverage Tool",
             "required": false,
             "groupName": "codeCoverage",
-            "defaultValue": "JaCoCo",
-            "helpMarkDown": "Select the tool with which summary files have been generated. Only Jacoco support available currently. Jacocoant.jar should be present in lib folder of Ant installation.",
+            "defaultValue": "NoCoverage",
+            "helpMarkDown": "Select the tool for evaluating code coverage. Only Jacoco support available currently. Jacocoant.jar should be present in lib folder of Ant installation.",
```

Comment:
if possible it would be useful to show the jar requirement in the UI as a text when the user selects Jacoco from the dropdown